features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/api-category/api-category-add/template.jag

```diff
@@ -0,0 +1,97 @@
+<% jagg.template("api-category/api-category-add", function(inputs, outputs, jagg) { %>
+
+    <%
+    var uuid = request.getParameter("uuid");
+    var action = request.getParameter("action");
+    var pageTitle = i18n.localize("Add API Category");
+    var nameFieldDisableStatus = "";
+
+    if(action == null){
```

Comment:
Formatting issue. There are other places as well. Please fix them all.

Reply:
fixed in https://github.com/wso2/carbon-apimgt/pull/8077/commits/c6c0ee00e2fc4d641567d19b696b59a2a92f077b